### ISSUE TITLE
Plugin won't work if enabled in config file at startup

### ISSUE THREAD

**Issue by etlfg** [matterpoll/matterpoll#317]
#### Summary
Can't use the plugin if already enabled manually in the config file at startup.

#### Steps to reproduce
1. Configure your config.json with at least the following block
```
  "PluginSettings": {
    "Enable": true,
    "Plugins": {
        "com.github.matterpoll.matterpoll": {
            "experimentalui": false,
            "trigger": "poll"
        }
    },
    "PluginStates": {
        "com.github.matterpoll.matterpoll": {
            "Enable": true
        }
    }
 }
```
2. Startup the mattermost server
3. Go to any channel and begin typing "/poll", nothing happens
4. Go to the admin section to turn off and on the plugin (Plugins > Plugin Management - Disable then Enable)
3. Go to any channel and begin typing "/poll", the dialog box pop in to help you complete

#### Expected behavior
The plugin should be up and running at startup.

#### Observed behavior (that appears unintentional)
A configuration change seems to be needed for the plugin to work.

#### Server configuration
**Operating system:**
Kubernetes v1.16.12+vmware.1 on top of Docker 18.9.9 on top of Ubuntu 16.04.6 LTS on top of 4.15.0-107-generic 

**Mattermost version:**
5.26.0

**Matterpoll version:**
1.3.0

**Updated from an older Matterpoll version or fresh install:**

Fresh

<details>
<summary>Mattermost server logs</summary>

```
N/A
```
</details>

#### Additional context

I don't know matterpoll code but I quickly found the configuration.go file where sits the main function [onConfigurationChange](https://github.com/matterpoll/matterpoll/blob/9ab0ecb7ebea9eb0b6cebb47855b5380bea8f605/server/plugi/configuration.go#L17)
Let me think the plugin only can be enabled if the enable parameter is set when running.

**Comment by mmasdoria** [comment]
Hi, same problem and if I restart the container I have to do same actions again. (Go to the admin section to turn off and on the plugin (Plugins > Plugin Management - Disable then Enable)
Please have a look to this issue.
Regards.

**Comment by hanzei** [comment]
@etlfg @mmasdoria Do your server logs contain any errors? Did you by an chance only tested using DMs?

**Comment by etlfg** [comment]
None. Thus the "N/A" below Mattermost server logs section.

Got `{"level":"info","ts":1598535970.5626068,"caller":"app/plugin.go:271","msg":"Syncing plugin from file store","bundle":"plugins/com.github.matterpoll.matterpoll.tar.gz"}` at startup. Once turned off and on, got `{"level":"info","ts":1598537268.8771267,"caller":"bleveengine/bleve.go:267","msg":"UpdateConf Bleve"}`. That's it. Very clean logs.

Could you explain what you mean by "DM" ?

**Comment by hanzei** [comment]
DM means direct message.

**Comment by etlfg** [comment]
Ok, then no, I observed the issue with both public channels and DMs.

**Comment by ewwollesen** [comment]
Just wanted to add that we just upgraded to 5.26.2 and we are having the same issue. Plugin was showing that it was enabled and looked good in the console but it didn't recognize the slash command. I disabled and then enabled it and it started working again.

As a side now, when I did the disable/enable I got probably 100+ of these messages but it is working now so I don't know.

`{"level":"error","ts":1600709658.3153188,"caller":"mlog/log.go:190","msg":"Access to route for non-existent plugin","missing_plugin_id":"com.github.matterpoll.matterpoll","error":"plugin not found: com.github.matterpoll.matterpoll"}`

**Comment by kaakaa** [comment]
I can reproduce this on [mattermost/mattermost\-preview:5\.26\.0](https://hub.docker.com/layers/mattermost/mattermost-preview/5.26.0/images/sha256-03d9893496553dc75d7d3f1f3d54301552ffa0f2a278b09ff2de47a0a494e03f?context=explore) image. For testing, I did same steps with [standup-raven](https://github.com/standup-raven/standup-raven), then standup-raven did not have this problem and I was able to use `/standup help` command immediately after startup.

So I supposed that this issue is related to #321. I've tested with the master branch build including fix of #321 , and in the few times I've tried, this issue doesn't seem to happen.

**Comment by dalenewman** [comment]
I can confirm I have this issue as well, Mattermost 6.4.2 and Matterpoll 1.3.0.

**Comment by mmasdoria** [comment]
Yes I also have the issue with Mattermost 6.4.0 and matterpoll 1.4.0.

**Comment by hanzei** [comment]
@kaakaa Per the report from @mmasdoria the bug also seems to be happening on Matterpoll 1.4.0. Could you check if you can still reproduce it?

**Comment by kaakaa** [comment]
I can't reproduce this issue on Mattermost 6.4.0 with Mattepoll v1.4.0, so this issue would be solved in Matterpost v1.4.0. If this happens with Matterpoll v1.4.0 or later, feel free to reopen this.

---

I have verified the following process to see if this problem can be reproduced on Matermost 6.4.0 with Matterpoll v1.3.0 and v1.4.0.

1. Upload and enable Matterpoll plugin from system console (and confirm that `/poll` command run successfully)
2. Restart Matermost server
3. Execute `/poll` command
4. Restart (disable and enable) Matterpoll plugin (and confirm that `/poll` command run successfully)

The most important point is whether `/poll` command run successfully at `3.`, and the results for each version were as follows.

  * **Matterpoll v1.3.0**: `/poll` return error `Command with a trigger of '/poll' not found. Click here to send as a message.`
  * **Matterpoll v1.4.0**: `/poll` run successfully

I tested this process twice, so Matterpoll v1.4.0 would resolve this issue.
